src/metadata/taglib_handler.cc

```diff
@@ -380,7 +380,7 @@ std::unique_ptr<IOHandler> TagLibHandler::serveContent(const std::shared_ptr<Cds
         auto f = TagLib::ASF::File(&roStream);
 
         if (!f.isValid())
-            throw_std_runtime_error("Could not open flac file: {}", item->getLocation().c_str());
+            throw_std_runtime_error("Could not open wma file: {}", item->getLocation().c_str());
```

Comment:
Whoops!

Reply:
Been searching for flac in the the code 😄